test/scalability/mzcompose.py

```diff
@@ -141,7 +141,8 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
     print(f"Baseline: {baseline_endpoint}")
     print(f"Other endpoints: {other_endpoints}")
 
-    # fetch git tags so that the commit messages of version tags can be resolved
+    # fetch git tags so that the commit messages of main and version tags can be resolved
+    git.fetch(branch="main")
```

Comment:
Can't this be merged into the `git fetch` below which includes tags? A single `git fetch origin --tags` should get the main branch and all tags already, right?

Looking at the source code of `git.fetch` I don't think `git.fetch(branch="main")` even works because of the `if branch and not remote` check.

Reply:
Oh yes, thank you